app/controllers/news_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
 
   private
   def set_section
-    @section = SectionHome.find_by!(slug: params[:section]).section
+    Rollbar.silenced do
```

Comment:
Having looked at the silenced block, I think it can be altered to only handle specific exceptions https://github.com/rollbar/rollbar-gem/blob/ffb006fa7bb900b82f76af7a3d127254dd9add25/lib/rollbar.rb#L89